### PR TITLE
Context slides

### PR DESCRIPTION
**Pull request by okonet**

### CODE REVIEW

File: slides/resilient-component-libraries.mdx [hunk 1]
```diff
@@ -412,7 +445,7 @@ export { DatePicker } from "material-ui"
 
 ---
 
-> ⏳ 30 minutes
+> 20 minutes
```
Comment (sapegin) [inline slides/resilient-component-libraries.mdx:383]: I liked the hourglass emoji 😢

File: slides/resilient-component-libraries.mdx [hunk 2]
```diff
@@ -987,6 +1026,35 @@ const RedBox = ({className, ...props}) => (
 
 ---
 
+```jsx
+import React, { useCallback } from 'react';
+import { useDropzone } from 'react-dropzone';
+
+function MyDropzone() {
+	const onDrop = useCallback(acceptedFiles => {
```
Comment (sapegin) [inline slides/resilient-component-libraries.mdx]:
```suggestion
	const onDrop = React.useCallback(acceptedFiles => {
```

For consistency with other slides?

File: slides/resilient-component-libraries.mdx [hunk 3]
```diff
@@ -987,6 +1026,35 @@ const RedBox = ({className, ...props}) => (
...
+	}, []);
+	const { getRootProps, getInputProps, isDragActive } = useDropzone({ onDrop });
+
+	return (
+		<div {...getRootProps()}>
+			<input {...getInputProps()} />
+			{isDragActive ? (
+				<p>Drop the files here ...</p>
+			) : (
+				<p>Drag 'n' drop some files here, or click to select files</p>
```
Comment (sapegin) [inline slides/resilient-component-libraries.mdx]:
```suggestion
				<p>Drag'n'drop some files here, or click to select files</p>
```

File: slides/resilient-component-libraries.mdx [hunk 4]
```diff
@@ -1125,19 +1202,63 @@ const RedBox = ({className, ...props}) => (
...
-<Figure
-	src={'/assets/component-driven.svg'}
-	alt="Component-driven Logo"
-	caption={
-		<a href="https://component-driven.io">https://component-driven.io</a>
-	}
-/>
+<Primary>
+
+# React.Context
```
Comment (sapegin) [inline slides/resilient-component-libraries.mdx]:
```suggestion
# React Context
```

File: slides/resilient-component-libraries.mdx [hunk 5]
```diff
@@ -1125,19 +1202,63 @@ const RedBox = ({className, ...props}) => (
...
+<ThemeProvider theme={theme}>
+	<App />
+</ThemeProvider>;
+```
+
+</SplitTitle>
+
+---
+
+## Avoid re-renders by passing React reconciliation
```
Comment (sapegin) [inline slides/resilient-component-libraries.mdx]: Bypassing? I'm not sure what “passing React reconciliation” means.

File: slides/resilient-component-libraries.mdx [hunk 6]
```diff
@@ -1125,19 +1202,63 @@ const RedBox = ({className, ...props}) => (
...
+
+[React at 60fps](https://medium.com/hackernoon/react-at-60fps-4e36b8189a4c)
+
+---
+
+<Demo src="http://react-sync-scroll.netlify.com" />
```
Comment (sapegin) [inline slides/resilient-component-libraries.mdx]:
```suggestion
<Demo src="https://react-sync-scroll.netlify.app/" />
```

File: slides/resilient-component-libraries.mdx (hunk 1 above)
Comment (okonet) [reply slides/resilient-component-libraries.mdx:383]: Me too but it looks ugly in italics

File: slides/resilient-component-libraries.mdx (hunk 5 above)
Comment (okonet) [reply slides/resilient-component-libraries.mdx]: yes! thanks!

File: slides/resilient-component-libraries.mdx [hunk 7]
```diff
@@ -1264,6 +1199,17 @@ import theme from '../theme';
 
 ---
 
+## React.Context
+
+- global state store (you don't need Redux!)
```
Comment (sapegin) [inline slides/resilient-component-libraries.mdx]:
```suggestion
- global state store (you may not need Redux!)
```

File: slides/resilient-component-libraries.mdx [hunk 8]
```diff
@@ -1264,6 +1199,17 @@ import theme from '../theme';
 
 ---
 
+## React.Context
```
Comment (sapegin) [inline slides/resilient-component-libraries.mdx]:
```suggestion
## React Context
```

**Review by sapegin — APPROVED** [review]

**Pull request merged**